### ISSUE TITLE
Unable to run Readme example

### ISSUE THREAD

**Issue by matt9704** [google-research/pegasus#36]
Hi! I run your work on google colab. At this step:
`!python3 pegasus/bin/train.py --params=aeslc_transformer \
--param_overrides=vocab_filename=ckpt/pegasus_ckpt/c4.unigram.newline.10pct.96000.model \
--train_init_checkpoint=ckpt/pegasus_ckpt/model.ckpt-1500000 \
--model_dir=ckpt/pegasus_ckpt/aeslc`
The error comes up.
`Traceback (most recent call last):
  File "pegasus/bin/train.py", line 17, in <module>
    from pegasus.data import infeed
  File "/usr/local/lib/python3.6/dist-packages/pegasus/__init__.py", line 1, in <module>
    from pegasus.parser import *
  File "/usr/local/lib/python3.6/dist-packages/pegasus/parser.py", line 10, in <module>
    from pegasus.rules import _build_rule, ParseError, Lazy
  File "/usr/local/lib/python3.6/dist-packages/pegasus/rules.py", line 62
    print 'pegasus: {}\x1b[2;38;5;241menter {} -> {}\x1b[m'.format(depth, repr(char()), _name)
                                                          ^
SyntaxError: invalid syntax`

**Comment by JingqingZ** [comment]
It seems you installed the wrong [pegasus by pip](https://pypi.org/project/pegasus/). The steps described in https://github.com/google-research/pegasus/issues/16 may help you set up in colab.

**Comment by matt9704** [comment]
Thanks

**Comment by sulata2** [comment]
> Hi! I run your work on google colab. At this step:
> `!python3 pegasus/bin/train.py --params=aeslc_transformer \ --param_overrides=vocab_filename=ckpt/pegasus_ckpt/c4.unigram.newline.10pct.96000.model \ --train_init_checkpoint=ckpt/pegasus_ckpt/model.ckpt-1500000 \ --model_dir=ckpt/pegasus_ckpt/aeslc`
> The error comes up.
> `Traceback (most recent call last): File "pegasus/bin/train.py", line 17, in <module> from pegasus.data import infeed File "/usr/local/lib/python3.6/dist-packages/pegasus/__init__.py", line 1, in <module> from pegasus.parser import * File "/usr/local/lib/python3.6/dist-packages/pegasus/parser.py", line 10, in <module> from pegasus.rules import _build_rule, ParseError, Lazy File "/usr/local/lib/python3.6/dist-packages/pegasus/rules.py", line 62 print 'pegasus: {}\x1b[2;38;5;241menter {} -> {}\x1b[m'.format(depth, repr(char()), _name) ^ SyntaxError: invalid syntax`

I am getting the same issue and if i do with pip install -e pegagus then tensorflow says to upgrage the version and the same issye with pegagus.. any help ?

**Comment by sulata2** [comment]
> Hi! I run your work on google colab. At this step:
> `!python3 pegasus/bin/train.py --params=aeslc_transformer \ --param_overrides=vocab_filename=ckpt/pegasus_ckpt/c4.unigram.newline.10pct.96000.model \ --train_init_checkpoint=ckpt/pegasus_ckpt/model.ckpt-1500000 \ --model_dir=ckpt/pegasus_ckpt/aeslc`
> The error comes up.
> `Traceback (most recent call last): File "pegasus/bin/train.py", line 17, in <module> from pegasus.data import infeed File "/usr/local/lib/python3.6/dist-packages/pegasus/__init__.py", line 1, in <module> from pegasus.parser import * File "/usr/local/lib/python3.6/dist-packages/pegasus/parser.py", line 10, in <module> from pegasus.rules import _build_rule, ParseError, Lazy File "/usr/local/lib/python3.6/dist-packages/pegasus/rules.py", line 62 print 'pegasus: {}\x1b[2;38;5;241menter {} -> {}\x1b[m'.format(depth, repr(char()), _name) ^ SyntaxError: invalid syntax`

how you fixed this issue ?

**Comment by sulata2** [comment]
> It seems you installed the wrong [pegasus by pip](https://pypi.org/project/pegasus/). The steps described in #16 may help you set up in colab.

I tried using as per #16 , but still i get the above issue invalid syntax (pegasus /rules.py). if i use (!pip3 install -e pegasus) ,it is asking to upgrade tensorflow and i upgrade tensorflow to 2.0 . then i get this syntax error.

**Comment by JingqingZ** [comment]
The `pegasus` that you have installed by `pip install -e pegasus` is **NOT** this `pegasus`. You will need to git clone the source code and then pip install.

**Comment by sulata2** [comment]
> The `pegasus` that you have installed by `pip install -e pegasus` is **NOT** this `pegasus`. You will need to git clone the source code and then pip install.

I did git clone first then at end pip install pegagus ,but getting as above error .let me try again.

**Comment by sulata2** [comment]
> The `pegasus` that you have installed by `pip install -e pegasus` is **NOT** this `pegasus`. You will need to git clone the source code and then pip install.

I am running with tensorflow 2.2.0 since pegagus is not accepting tensorflow 1.15.2 version ,so taking higher version of tensorflow. but getting the error "dist-packages/pegasus/rules.py", line 62 print 'pegasus: {}\x1b[2;38;5;241menter {} -> {}\x1b[m'.format(depth, repr(char()), _name) ^ SyntaxError: invalid syntax"

**Comment by sulata2** [comment]
I tried as you suggested .. after cloning github , then pip install pegasus still getting same issue .
!python3 pegasus/bin/train.py --params=aeslc_transformer \
--param_overrides=vocab_filename=ckpt/pegasus_ckpt/c4.unigram.newline.10pct.96000.model \
--train_init_checkpoint=ckpt/pegasus_ckpt/model.ckpt-1500000 \
--model_dir=ckpt/pegasus_ckpt/aeslc

Traceback (most recent call last):
  File "pegasus/bin/train.py", line 17, in <module>
    from pegasus.data import infeed
  File "/usr/local/lib/python3.7/dist-packages/pegasus/__init__.py", line 1, in <module>
    from pegasus.parser import *
  File "/usr/local/lib/python3.7/dist-packages/pegasus/parser.py", line 10, in <module>
    from pegasus.rules import _build_rule, ParseError, Lazy
  File "/usr/local/lib/python3.7/dist-packages/pegasus/rules.py", line 62
    print 'pegasus: {}\x1b[2;38;5;241menter {} -> {}\x1b[m'.format(depth, repr(char()), _name)
                                                          ^
SyntaxError: invalid syntax

**Comment by JingqingZ** [comment]
> /usr/local/lib/python3.7/dist-packages/pegasus/rules.py

Again, your code is calling the pegasus that is not ours. Please make sure the other pegasus is uninstalled.

**Comment by sulata2** [comment]
> > /usr/local/lib/python3.7/dist-packages/pegasus/rules.py
> 
> Again, your code is calling the pegasus that is not ours. Please make sure the other pegasus is uninstalled.

Thank you for your guidance. one question : as per the readme link , i saw pegasus setuptools.setup(
    name='pegasus',
    version='0.0.1', ....

but when i try to install this version of pegagus , i am getting message this version not available but 
ERROR: Could not find a version that satisfies the requirement pegasus==0.0.1 (from versions: 0.1.0, 0.1.1, 0.1.2, 0.1.3)
ERROR: No matching distribution found for pegasus==0.0.1  .  so facing issue.. which version i should install ?

**Comment by sulata2** [comment]
Thank you . I got that pegasus 0.0.1 worked fine. I changed in setup.py(because in setup.py it was having earlier  and ran with !pip install -e pegasus . worked. but facing some other tensorflow issue.ModuleNotFoundError: No module named 'tensorflow.contrib'.

**Comment by vzagde** [comment]
I am too getting the same error I installed pegasus using pip3 command and all the TensorFlow dependencies with the updated version, Not able to understand how I can resolve this
